features/activation.feature

```diff
@@ -1,9 +1,38 @@
 @slow_process
 Feature: Test product activation
 
+  @skip-sles-15
   Scenario: System registration
     Given I have a system with activated base product
 
+    Then the output should contain "Activating SLES"
+    And the output should contain "Adding zypper service"
+    And the output should contain "Installing release package"
```

Comment:
But it isn't supposed to install the release package for the base product? :thinking:

Reply:
yes I already fixed this